lib/src/utils/kratos_message.dart

```diff
@@ -1,178 +1,1398 @@
+  const ErrorValidationSuchNoWebAuthnUser();
+
+  static const id = 4000015;
+}
+
+final class ErrorValidationLookupInvalid extends KratosMessage {
+  const ErrorValidationLookupInvalid();
+
+  static const id = 4000016;
+}
+
+final class ErrorValidationMaxLength extends KratosMessage {
+  const ErrorValidationMaxLength();
+
+  static const id = 4000017;
+}
+
+final class ErrorValidationMinimum extends KratosMessage {
+  const ErrorValidationMinimum();
+
+  static const id = 4000018;
+}
+
+final class ErrorValidationExclusiveMinimum extends KratosMessage {
+  const ErrorValidationExclusiveMinimum();
+
+  static const id = 4000019;
+}
+
+final class ErrorValidationMaximum extends KratosMessage {
+  const ErrorValidationMaximum();
+
+  static const id = 4000020;
+}
+
+final class ErrorValidationExclusiveMaximum extends KratosMessage {
+  const ErrorValidationExclusiveMaximum();
+
+  static const id = 4000021;
+}
+
+final class ErrorValidationMultipleOf extends KratosMessage {
+  const ErrorValidationMultipleOf();
+
+  static const id = 4000022;
+}
+
+final class ErrorValidationMaxItems extends KratosMessage {
+  const ErrorValidationMaxItems();
+
+  static const id = 4000023;
+}
+
+final class ErrorValidationMinItems extends KratosMessage {
+  const ErrorValidationMinItems();
+
+  static const id = 4000024;
+}
+
+final class ErrorValidationUniqueItems extends KratosMessage {
+  const ErrorValidationUniqueItems();
+
+  static const id = 4000025;
+}
+
+final class ErrorValidationWrongType extends KratosMessage {
+  const ErrorValidationWrongType({
+    required this.actualType,
+    required this.allowedTypesList,
+  });
+
+  factory ErrorValidationWrongType.fromContext(
+    ContextParameters? contextParameters,
+  ) {
+    return ErrorValidationWrongType(
+      actualType: contextParameters!['actual_type']!,
+      allowedTypesList:
+          commaSepratedString(contextParameters['allowed_types_list']),
+    );
+  }
+
+  final String actualType;
+  final String allowedTypesList;
+
+  static const parameterNames = ['actual_type'];
+
+  static const id = 4000026;
+}
+
+final class ErrorValidationDuplicateCredentialsOnOIDCLink
+    extends KratosMessage {
+  const ErrorValidationDuplicateCredentialsOnOIDCLink();
+
+  static const id = 4000027;
+}
+
+final class ErrorValidationCredentialAlreadyUsedByAnotherAccount
+    extends KratosMessage {
+  const ErrorValidationCredentialAlreadyUsedByAnotherAccount({
+    required this.credentialIdentifierHint,
+    required this.availableCredentialTypesList,
+  });
+
+  factory ErrorValidationCredentialAlreadyUsedByAnotherAccount.fromContext(
+    ContextParameters? contextParameters,
+  ) {
+    return ErrorValidationCredentialAlreadyUsedByAnotherAccount(
+      credentialIdentifierHint:
+          contextParameters!['credential_identifier_hint']!,
+      availableCredentialTypesList: commaSepratedString(
+        contextParameters['available_credential_types_list'],
+      ),
```

Comment:
Let's not pre-format the lists, the message models should store the actual list itself